### ISSUE TITLE
SQS Golang example is outdated

### ISSUE THREAD

**Issue by glebarez** [yandex-cloud/docs#707]
The example: https://cloud.yandex.ru/ru/docs/message-queue/instruments/golang doesn't work with recent versions of AWS SQS SDK.
Recently SQS migrated their service's protocol from the Query protocol which was XML based, to a JSONRPC protocol, and updated the default SDK behavior accordingly.

See https://github.com/aws/aws-sdk-go-v2/issues/2451 for exact error and explanation.

**Comment by glebarez** [comment]
The workaround is to downgrade corresponding deps:
```
github.com/aws/aws-sdk-go-v2 v1.22.2
github.com/aws/aws-sdk-go-v2/service/sqs v1.26.0
```

**Comment by pyatnitsev** [comment]
It seems fixed in #725
